src/widgets/index/index.ts

```diff
@@ -212,6 +216,7 @@ const index = (props: IndexProps): Index => {
             widget.init({
               helper: helper!,
               parent: this,
+              uiState: {},
```

Comment:
The `initialUiState` is only provided on the first search. Dynamically added widgets ignore the option.

Reply:
By this you mean that it's only used by the `init` of the index itself, right?

Reply:
I mean, only on the first loop over the `init` lifecycle. An `index` dynamically added won't take into account the `initialUiState`. We can provide it since we have access to the `instantsearchInstance` but I'm not sure which behavior is the one expected: ignored or taken into account?